lib/test.dart

```diff
@@ -39,6 +38,8 @@ export 'src/frontend/throws_matchers.dart';
 export 'src/frontend/timeout.dart';
 export 'src/frontend/utils.dart';
 
+import 'package:path/path.dart' as p;
```

Comment:
why did this move?

Reply:
just accidental I think, was playing around with a few things and we don't have the import order lint I guess